doc/developers/contributing.rst

```diff
@@ -678,54 +672,6 @@ To test code coverage, you need to install the `coverage
 
 3. Loop.
 
-Developers web site
--------------------
-
-More information can be found on the `developer's wiki
-<https://github.com/scikit-learn/scikit-learn/wiki>`_.
-
-
-Issue Tracker Tags
-------------------
```

Comment:
Also chose to remove this one: the core dev are the ones setting the tags, not the contributors.

Reply:
As long as the ones that are important for the contributors to understand are visible...

Reply:
I'm not sure what you mean Joel. Should I revert?

Reply:
The point is that here this explains the meaning of the tags to the developers. I think it's useful.

Reply:
Ok, reverting